app/assets/stylesheets/_forms.scss

```diff
@@ -23,4 +23,21 @@
   &:checked {
     background-color: $greige-darkest;
   }
+}
+
+.form-control.fancy_color {
+  border: none;
+  color: inherit!important;
+  background-color: inherit!important;
+}
```

Comment:
Just a consistent code style nitpick - add blank lines between your CSS rule sets and a space before your `!important`s.